<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rsa_cipher</class>
 <widget class="QDialog" name="rsa_cipher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_encrypt">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PlainText</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RSA_Cipher</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CipherText</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>130</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Infomation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>280</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>signture</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_info">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>150</y>
     <width>521</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_decrypt">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gen_keys">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Genergate keys</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_sign">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>300</y>
     <width>521</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_cipher_text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>471</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_plain_text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>471</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_verify">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Vefity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sign">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
